src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -205,6 +209,11 @@ const VisitorSpaceSearchPage: FC = () => {
 		true
 	);
 
+	// Calculate when to show the manyResultsTile
+	const resultsHasMorethanMaxItems = searchResults && searchResults?.total > RESULTS_MAX;
+	const showManyResultsTile =
+		resultsHasMorethanMaxItems && query.page === PAGE_NUMBER_OF_MANY_RESULTS_TILE;
```

Comment:
Doesn't this do the same thing?
```suggestion
	// Calculate when to show the manyResultsTile
	const showManyResultsTile = query.page === PAGE_NUMBER_OF_MANY_RESULTS_TILE;
```